avm/client/build_wasm.sh

```diff
@@ -1,9 +1,13 @@
-#!/bin/sh
-(
-  cd ../../air-interpreter;
-  marine build  --release --features marine
-)
+#! /usr/bin/env bash
 
-mkdir -p ./dist/
-cp ../../target/wasm32-wasi/release/air_interpreter_server.wasm ./dist/avm.wasm
+if [[ -f dist/avm.wasm ]]; then
```

Comment:
That would prevent rebuilding after `cargo clean`, afaiu. Because `cargo clean` won't remove `dist/avm.wasm`.

I'm not sure it is a problem, cuz I'm not sure how `avm/client/build_wasm.sh` is used by humans, if at all.